tools/test.py

```diff
@@ -28,6 +28,17 @@ def parse_args():
     parser.add_argument('config', help='test config file path')
     parser.add_argument('checkpoint', help='checkpoint file')
     parser.add_argument('--out', help='output result file')
+    out_options = ['class_scores', 'pred_score', 'pred_label', 'pred_class']
+    parser.add_argument(
+        '--out-items',
+        nargs='+',
+        default=['all'],
```

Comment:
'all' and 'none' is unneccessal, just set default = out_options, nargs='*'
if user want to output all, just use default.
if user want to output none, the output_file will be empty. Unless there is an error, it will not appear, when he set "--out xxx.pkl"
and shen use '----out-items "  and add nothing behind it, a warning need to raise.

Reply:
Using `none` and `all` is a more intuitive way. `all` is a shortcut to all options, and `none` can explicitly tell users all options are disabled.
As for possible empty output files, unless users don't specify `--metrics` and explicitly specify `--out-items none`, it won't happen. I think that's expected behavior.